doc/source/analyzing/objects.rst

```diff
@@ -625,6 +637,34 @@ If an axis is provided, it will project along that axis and return it to you:
    rho_proj = reg.mean(("gas", "temperature"), axis="y", weight=("gas", "density"))
    rho_proj.plot()
 
+You can also compute the ``std`` (standard deviation), which accepts a field,
+axis, and weight function. If the axis is not specified, it will
+return the standard deviation of the specified field, weighted by the weight
+argument.  The weight argument defaults to ``ones``. For instance:
+
+.. code-block:: python
+
+   std_rho = reg.std(("gas", "density"))
+   std_rho_by_vol = reg.std(("gas", "density"), weight=("gas", "cell_volume"))
+
+This is equivalent to:
+
+.. code-block:: python
+
+   std_rho = reg.quantities.weighted_standard_deviation(
+       ("gas", "density"), weight_field=("index", "ones")
+   )
+   std_rho_by_vol = reg.quantities.weighted_standard_deviation(
+       ("gas", "density"), weight_field=("gas", "cell_volume")
+   )
+
+If an axis is provided, it will project along that axis and return it to you:
+
+.. code-block:: python
+
+   vy_std = reg.std(("gas", "velocity_y"), axis="y", weight=("gas", "density"))
+   vy_std.plot()
```

Comment:
this paragraph is so similar to the previous one, I think we should merge them

Reply:
Ehh. I agree that they are very similar (in fact I copied, pasted, and edited to get this paragraph). But I feel like there are competing interests here. The first and second moments are of course different, and `std` was not documented separately prior to this (an oversight I assume), so I think we should leave it as-is.

Reply:
ok seems fair